DigitalLearningSolutions.Data.Tests/TestHelpers/TutorialContentTestHelper.cs

```diff
@@ -96,5 +97,15 @@ FROM aspProgress
                 new { progressId, tutorialId }
             ).FirstOrDefault();
         }
+
+        public IEnumerable<int> GetAspProgressFromTutorialId(int tutorialId)

```

Comment:
With this name I would assume we are returning whole records. Could we specify this is just the AspProgressId? Depending on how the other comment in the ProgressDataService is tackled, this might need to be refactored anyway.